libsolidity/analysis/PostTypeChecker.cpp

```diff
@@ -363,6 +363,70 @@ struct NoVariablesInInterfaceChecker: public PostTypeChecker::Checker
 	/// Flag indicating whether we are currently inside a StructDefinition.
 	int m_insideStruct = 0;
 };
+
+struct FreeFunctionDependencies: public PostTypeChecker::Checker
+{
+	FreeFunctionDependencies(ErrorReporter& _errorReporter):
+		Checker(_errorReporter)
+	{}
+	bool visit(ContractDefinition const& _contract) override
```

Comment:
This could use a bit more whitespace:

```suggestion
	FreeFunctionDependencies(ErrorReporter& _errorReporter):
		Checker(_errorReporter)
	{}
	
	bool visit(ContractDefinition const& _contract) override
```

Same for other definitions in this struct.